lib/search_solr_tools/logging/sst_logger.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+require 'logging'
+require 'search_solr_tools'
```

Comment:
I'm apparently losing my mind. `search_solr_tools.rb` requires `search_solr_tools/logging/sst_logger` -- is this a circular situation or am I completely missing a different `search_solr_tools` that's being required here?

Reply:
I could probably have this just manually require the configuration itself directly.

Ruby seems to handle the circular requires just fine, but I think it would be cleaner to modify this since the logger itself doesn't need anything in the main search_solr_tools file except the configuration stuff.

Reply:
It's a red flag to me.

Reply:
What "configuration stuff" are you referring to?

Reply:
@scott-lewis-nsidc I see a reference to `SearchSolrTools::SolrEnvironments` in the file `bin/search_solr_tools`, but the `require` statement should be referring to `lib/search_solr_tools.rb`.

Reply:
I've modified the require here to specifically have the logger only load the config/environment file rather than the whole search_solr_tools module.